packages/client/mutations/ToggleTeamDrawerMutation.ts

```diff
@@ -0,0 +1,73 @@
+import {DrawerTypes} from 'parabol-client/types/constEnums'
+import graphql from 'babel-plugin-relay/macro'
+import {commitMutation} from 'react-relay'
+import {StandardMutation} from '../types/relayMutations'
+import toTeamMemberId from '../utils/relay/toTeamMemberId'
+import {ToggleTeamDrawerMutation as TToggleTeamDrawerMutation} from '../__generated__/ToggleTeamDrawerMutation.graphql'
+
+graphql`
+  fragment ToggleTeamDrawerMutation_teamMember on ToggleTeamDrawerSuccess {
+    hideAgenda
+    hideManageTeam
+  }
+`
+
+const mutation = graphql`
+  mutation ToggleTeamDrawerMutation($teamId: ID!, $teamDrawerType: TeamDrawer!) {
+    toggleTeamDrawer(teamId: $teamId, teamDrawerType: $teamDrawerType) {
+      ... on ErrorPayload {
+        error {
+          message
+        }
+      }
+      ...ToggleTeamDrawerMutation_teamMember @relay(mask: false)
+    }
+  }
+`
+
+const ToggleTeamDrawerMutation: StandardMutation<TToggleTeamDrawerMutation> = (
+  atmosphere,
+  variables,
+  {onError, onCompleted}
+) => {
+  return commitMutation<TToggleTeamDrawerMutation>(atmosphere, {
+    mutation,
+    variables,
+    updater: (store) => {
```

Comment:
if the mutation returns a team object you can completely remove the updater. yay less code!